config/application.yml.default

```diff
@@ -102,7 +102,7 @@ idv_public_key: 'LS0tLS1CRUdJTiBQVUJMSUMgS0VZLS0tLS0KTUZ3d0RRWUpLb1pJaHZjTkFRRUJ
 idv_private_key: 'LS0tLS1CRUdJTiBSU0EgUFJJVkFURSBLRVktLS0tLQpNSUlCT2dJQkFBSkJBS3p4d25rbUxqeGx1NmhsRlQ2d2JreUlweHNtYkMyaGNhblMwaG5abUNEYitMSFowTnNBCjdHWURpZkMxQlRBMHRuRFo0Zm9HNTRmYjNzYk9ubGpGWXVNQ0F3RUFBUUpCQUp6TUMvOSs2RWlHQzkrZTFlWWkKVzc0ejN4MjBkanZndFlhOHh4UDh2ZnA3TjdKQXMvaGNUbjVLOCtDM2swaXUyR2RNb21qSlp2ckxwT0IyTWh4RQo3QkVDSVFEVERhbVRCMHhKSlVpV0ljNk15Y0dFa2J4SEZ3eEtURVNCaHhzREFISDZEUUloQU5IR2NwVUs5dmVSCkdrZlZTOS9MSVNZQlk2YzRUZk1NUFJZU21KVHFNRVN2QWlBZFdiY05aV1JzZjZ6YWhCVVBhemRvVWtRV3R0UFUKdVVxRm9ONVd5b2NQT1FJZ1FrUjlaK1haMUtVcTl5eERWc1FWaWFzQXJ3K1RXRWN5ZU9tUTkrSHZNNU1DSUcrMQpxVldqNW9PL0FBSU1QbXZVZmp5L0JnMnhEQVRiOEp6alFrQ3dLSnNwCi0tLS0tRU5EIFJTQSBQUklWQVRFIEtFWS0tLS0tCg=='
 idv_send_link_attempt_window_in_minutes: 10
 idv_send_link_max_attempts: 5
-in_person_proofing_enabled: true
+in_person_proofing_enabled_issuers: '[null]'
```

Comment:
why add the null in there? would it be easier to default to empty array?
```suggestion
in_person_proofing_enabled_issuers: '[]'
```

Reply:
The intention is for `[null]` and `[]` to be meaningfully different:

- `[]` disallows support for everyone
- `[null]` would allow requests not associated with a service provider (`current_sp&.issuer == nil`)

Reply:
oh I see.... that's sneaky... I have never seen us rely on the null/nil issue. I like that it works but hope it's clear enough to future us!

Reply:
Yeah, I also contemplated a couple other ideas:

- Allow a wildcard value (`["*"]`) which allows everyone
   - Unclear if we'd want to distinguish between _any SP_ and _no SP_
- Let the configuration be either a boolean or array of issuers
   - Not supported by `IdentityConfig` and I imagine it's not something we'd want to support
- Separate configuration values

Reply:
I do like the `wildcard` option or separate configs a little more. That seems like it will be a little more obvious to our future selves

Reply:
Okay, thinking something like this then?

- `in_person_proofing_enabled` (Boolean): Whether in-person proofing option is enabled, and specifically whether it's shown for sessions not associated with a service provider
- `in_person_proofing_enabled_issuers` (String\[\]): For proofing sessions which are associated with a service providers, assuming `in_person_proofing_enabled` is true, whether the option should be shown for that service provider's issuer.

Reply:
I like that approach. And it makes sense to me that enabling it globally would involve removing the feature flag. We need to get better about going back and cleaning those up anyway.